com.unity.render-pipelines.high-definition/Editor/Material/Lit/ShaderGraph/LitPass.template

```diff
@@ -330,6 +330,11 @@ Pass
 
         $SurfaceDescription.Emission: builtinData.emissiveColor = surfaceDescription.Emission;
 
+        //Note this will not fully work on transparent surfaces (can check with _SURFACE_TYPE_TRANSPARENT define)
+        //We will always overwrite vt feeback with the nearest. So behind transparent surfaces vt will not be resolved
+        //This is a limitation of the current MRT approach.
+        $SurfaceDescription.VTPackedFeedback: builtinData.vtPackedFeedback = surfaceDescription.VTPackedFeedback;
```

Comment:
Also:  why do some of the template files check if !defined(_SURFACE_TYPE_TRANSPARENT) and others don't?